src/frontend/EdFi.DataManagementService.Frontend.AspNetCore/Infrastructure/LoggingMiddleware.cs

```diff
@@ -41,20 +43,33 @@ public async Task Invoke(HttpContext context, ILogger<LoggingMiddleware> logger)
                 )
             );
 
+            FailureResponse failureResponse;
+
+            var validationErrors = new Dictionary<string, string[]>();
+
+            var value = new List<string>
+            {
+                ex.Message
+            };
+            validationErrors.Add("$.", value.ToArray());
+
+            failureResponse = FailureResponse.ForDataValidation(
+                "Data validation failed. See 'validationErrors' for details.",
+                validationErrors,
+                new List<string>().ToArray()
+            );
+
+            var options = new JsonSerializerOptions
+            {
+                Encoder = JavaScriptEncoder.UnsafeRelaxedJsonEscaping
+            };
```

Comment:
If we have another frontend, like a Lambda function, it will also require this same logic.  Perhaps it would make sense to move this logic to the `FailureResponse` class?

Reply:
Also, any idea why we are using `UnsafeRelaxedJsonEscaping`? It is in fact "unsafe". If we really do need it, please move this to a static field - no need to create every time.